### DISCUSSION TITLE
Can ArchR calculate gene score matrix only with peak counts matrix?

### DISCUSSION THREAD

**Discussion by Telogen** [GreenleafLab/ArchR#1191] [Questions / Documentation]
Hi, I 'm trying to calculate gene score matrix using ArchR, but I do not have neither fragment file nor BAM file. I only have the peak counts matrix, and this matrix has already pass the quality control and peak calling. Can I calculate gene score matrix only with this peak counts matrix using ArchR? Thanks!

**Comment by rcorces** [comment]
Please note that Issues are for bugs and error reports. This is a usage question which belongs in Discussions. I am transferring this there.

**Comment by rcorces** [comment]
ArchR must have either BAM or fragment files as input. A peak matrix is not suitable. No analysis (including gene score matrix calculation) can be performed from a peak matrix.
